Offseason2022/src/main/java/frc/robot/commands/Climber0Stow.java

```diff
@@ -15,20 +24,30 @@
  */
 public class Climber0Stow extends SequentialCommandGroup
 {
-  public Climber0Stow(Climber climber, Intake intake, FloorConveyor fConv, TowerConveyor tConv, Shooter shooter)
+  public Climber0Stow(Climber climber, Intake intake, FloorConveyor fConv, TowerConveyor tConv, Shooter shooter,
+      Drivetrain drivetrain)
   {
+    setName("Climber0Stow");
+
     addCommands(
-    // Add Commands here:
-    // Also add parallel commands using the
-    //
-    // addCommands(
-    // new command1(argsN, subsystem),
-    // parallel(
-    // new command2(argsN, subsystem),
-    // new command3(argsN, subsystem)
-    // )
-    // );
+        // Add Commands here:
 
+        // @formatter:off
+      new DriveSlowMode(drivetrain, false), 
+      new IntakeDeploy(intake, false), 
+      new IntakeRun(intake, INMode.INTAKE_STOP),
+      new FloorConveyorRun(fConv, FCMode.FCONVEYOR_STOP), 
+      new TowerConveyorRun(tConv, TCMode.TCONVEYOR_STOP),
+      new ShooterRun(shooter, Mode.SHOOTER_STOP),
+      new ParallelDeadlineGroup(
+          new WaitUntilCommand(climber::moveClimberDistanceIsFinished),
+          new ClimberMoveToHeight(climber, Height.STOW_HEIGHT)),

```

Comment:
For consistency with other commands, let's put the closing (second) paren on this line on its own line. Kind of like closing brackets.